### ISSUE TITLE
Langstroth Stack does not appear to take world time speed into account

### ISSUE THREAD

**Issue by AranaiRa** [VinterNacht/FromGoldenCombs#5]
When doing a test to see how Langstroth Stacks work in Creative, I cranked up the world speed. It took about six months for the bees to go from "searching for flowers" to "out gathering", though the amount of real world time seemed reasonable for normal play.

**Comment by VinterNacht** [comment]
I'll have to take a look at that.  So when you didn't crank up the world speed, it worked as expected? But if you crank up the world speed, it behaves as though you haven't?

**Comment by AranaiRa** [comment]
Correct. The amount of time it takes for the hive to go through stages seems to be independent of the world speed.

**Comment by VinterNacht** [comment]
I remember noticing this once before, and I think there might be an explanation in the form of:
"When the hive checks to see if there are flowers, there are conditions it looks for. This includes the time of day and the weather.  If you have the time ramped up it may keep skipping this check."

I believe this behavior can be observed in vanilla Skeps as well. Either way, I'll look into it.  While it isn't fixed in the current rendition, I'll see if it can be and... to be honest... if it's worth the effort.
